### ISSUE TITLE
Implement stable 'channels' for the aspnet runtime and dotnet-install.ps1

### ISSUE THREAD

**Issue by natemcmaster** [aspnet/Universe#1283]
Now that 2.1 is RTM, we should implement channels to match the .NET Core runtime and SDK for dotnet-install.ps1/sh. For example, it's reasonable to expect this to work:
```
dotnet-install -Channel 2.1 -Architecture "x64" -Runtime aspnetcore -NoPath -Verbose
```

**Requirements**
Create a blob here: dotnetcli.blob.core.windows.net/dotnet/aspnetcore/Runtime/2.1/latest.version. This should be a simple text file which contains full version information for the latest stable RTM version of the runtime. Example:
```
2cd777f06183bf5639578ff3a4162cd3f576e989
2.1.2
```

This file should be updated for each servicing release of 2.1

**Comment by tonycoelho** [comment]
@natemcmaster any updates on when this will be supported/available?

**Comment by natemcmaster** [comment]
No plans to do this right now. Currently backlogged because we believe this is low priority. We can adjust this as we receive more feedback.

**Comment by natemcmaster** [comment]
My recommendation is to specify the exact version you want.
```
dotnet-install.ps1 -Version 2.1.4 -Runtime aspnetcore
```

**Comment by tonycoelho** [comment]
@natemcmaster The main advantage of specifying a channel is that it determines the latest version automatically. Specifying the -Version param on the script negates this feature. If support for the 2.1 channel is not going to be added anytime soon, can you provide an alternative mechanism for programmatically determining what the latest version of 2.1 is?

**Comment by natemcmaster** [comment]
> determines the latest version automatically.

I don't have an alternative for this. cc @leecow can you comment on this?

**Comment by chrishiestand** [comment]
I agree with @tonycoelho but in the meantime the hardcoded `-version` might be good enough for me in the meantime, thanks @natemcmaster.

Is there something we could `curl` first to determine what the latest supported version is? That might be an okay workaround for automated scripts.

**Comment by natemcmaster** [comment]
> Is there something we could curl first to determine what the latest supported version is? T

That's exactly the problem and why we haven't implemented this. Our deployment automation is not currently configured to maintain an endpoint consistently whenever we roll out 2.1.x updates. It's not a lot of work to figure it out once, or to do it manually. But making it work consistently for every release is going to take some effort.

**Comment by leecow** [comment]
I'll be merging a set of release metadata files to the core repo early next week and which will help with this. See https://github.com/dotnet/core/pull/1960.

**Comment by tonycoelho** [comment]
@leecow So what you're suggesting is to download the json file linked below and use it to determine what the latest version of the .NET Core runtime is correct? Is yes, is this file going to be maintained in the near future or is there another release json file that is more reliable?

https://github.com/dotnet/core/blob/new-json/release-notes/releases-index.json

**Comment by leecow** [comment]
@tonycoelho - yes, that's the suggestion and the set of files ([index](https://github.com/dotnet/core/blob/new-json/release-notes/releases-index.json) and [release channel](https://github.com/dotnet/core/blob/new-json/release-notes/2.1/releases.json) details) will be maintained, as the current version has been maintained.

**Comment by natemcmaster** [comment]
@livarcocc @johnbeisner - what do you think about updating dotnet-install.ps1/sh to use these new JSON files when `-Channel` is specified?

**Comment by tonycoelho** [comment]
> yes, that's the suggestion and the set of files will be maintained, as the current version has been maintained.

@leecow, great thanks, this is very helpful since we can use them to determine the latest version for each release. Will these be moved to master at some point?

**Comment by livarcocc** [comment]
@natemcmaster I think that makes the most sense actually. @johnbeisner can you please look into it.

**Comment by leecow** [comment]
@tonycoelho - Yes, I'm hoping to merge to master this coming Monday or Tuesday.

**Comment by aspnet-hello** [comment]
This issue was moved to aspnet/AspNetCore#3646
